src/js/fable-core/ListClass.js

```diff
@@ -15,13 +15,13 @@ function compareList(self, other) {
   }
 }
 
-export class List {
+export default class List {
   constructor(head, tail) {
     this.head = head;
     this.tail = tail;
   }
 
-  toString() {
+  ToString() {
```

Comment:
Maybe we need to change it but right now Fable 2 is [compiling `ToString` declaration with the first letter in lower case](https://github.com/fable-compiler/Fable/blob/c3a30aef66eebca2495706cbf19429dfc6021c16/src/dotnet/Fable.Compiler/Transforms/Fable2Babel.fs#L573-L575). The reasoning is this way JS debuggers will call `toString` if the prototype of an object declares it (though I haven't really checked that yet). In any case, thanks for fixing the custom `toString` implementation check in `Util/toString`, I forgot to do that 😅

Reply:
Ok, if JS Debuggers are the reason, I'll change it to camelCase.

Reply:
I just added a camelCase toString, this way you can do both if you decide to change.